temboardagent/api.py

```diff
@@ -1,41 +1,42 @@
 import time
-import pickle
-import base64
 
 from temboardagent.routing import add_route
 from temboardagent.errors import (HTTPError, SharedItem_exists,
-                            SharedItem_no_free_slot_left, SharedItem_not_found,
-                            NotificationError)
-from temboardagent.sharedmemory import Session, Command
-from temboardagent.types import *
-from temboardagent.tools import validate_parameters, hash_id
+                                  SharedItem_no_free_slot_left,
+                                  SharedItem_not_found,
+                                  NotificationError)
```

Comment:
My opinion is that it's better to have:
 - one import per row,
 - add a line break after the opening brace,
 - put a coma at the end of each line even the last one.